tests/src/main/java/net/neoforged/neoforge/debug/client/ClientTests.java

```diff
@@ -81,6 +84,27 @@ static void keyMappingTest(final DynamicTest test) {
         });
     }
 
+    @TestHolder(description = "Tests that the NamespacedDirectoryLister only collects resources from the specified namespace")
```

Comment:
This test should be marked as enable by default for the fail/pass to actually work.